src/Raven.Server/Documents/Handlers/ExpirationHandler.cs

```diff
@@ -48,6 +49,33 @@ public async Task ConfigExpiration()
             await DatabaseConfigurations(ServerStore.ModifyDatabaseExpiration, "read-expiration-config", GetRaftRequestIdFromQuery());
         }
 
+        [RavenAction("/databases/*/refresh/config", "GET", AuthorizationStatus.ValidUser)]
+        public Task GetRefreshConfig()
+        {
+            using (Server.ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+            {
+                RefreshConfiguration refreshConfig;
+                using (var recordRaw = Server.ServerStore.Cluster.ReadRawDatabaseRecord(context, Database.Name))
+                {
+                    refreshConfig = recordRaw?.GetRefreshConfiguration();
+                }
+
+                if (refreshConfig != null)
+                {
+                    using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+                    {
+                        context.Write(writer, refreshConfig.ToJson());
+                    }
+                }
+                else
+                {
+                    HttpContext.Response.StatusCode = (int)HttpStatusCode.NotFound;
+                }
+            }
+            return Task.CompletedTask;
+        }
+        
```

Comment:
Can we move it to dedicated `RefreshHandler`?

Reply:
Below method as well `/databases/*/admin/refresh/config", "POST"`

Reply:
done